src/com/company/books/Book.java

```diff
@@ -0,0 +1,64 @@
+package com.company.books;
+
+
+import com.company.users.Author;
+import com.company.users.Publisher;
+
+public class Book {
+    protected static int idCounter = 0;
```

Comment:
[Fix] campurile trebuie sa fie private, chiar daca sunt intr-o clasa care va fi extinsa, aceste campuri se pot accesa si cu getteri/setteri. Daca spre exemplu, ai fi pus toate entitatile in acelasi pachet, cele de user ar fi putut modifica campurile de aici.